CSC322-Grinnell/mayflower-directory
# Fix tests

- Remove fixtures for frisbee and admin tables that were causing errors
- Also fix syntax error in one of the test files

# Testing
1. Run `rails db:migrate`
2. Run `rails test`
3. All tests should pass
